src/main/java/com/butbetter/storage/model/Address.java

```diff
@@ -0,0 +1,142 @@
+package com.butbetter.storage.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+public class Address {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id", insertable = false, updatable = false, nullable = false)
```

Comment:
please use @NotNull instead of using nullable = false.
@NotNull will not trigger the underlying db but nullable = false will.
-> @NotNull is better performing

https://www.baeldung.com/hibernate-notnull-vs-nullable

Reply:
@NotNull is not a Part of hibernate

Reply:
I dont see any diffent between @NotNull and nullable = false

Reply:
Even though both of them prevent us from storing null values in the database, they take different approaches.

As a rule of thumb, we should prefer the @NotNull annotation over the @Column(nullable = false) annotation. This way, we make sure the validation takes place before Hibernate sends any insert or update SQL queries to the database.

Also, it's usually better to rely on the standard rules defined in the Bean Validation, rather than letting the database handle the validation logic.

But, even if we let Hibernate generate the database schema, it'll translate the @NotNull annotation into the database constraints. We must then only make sure that hibernate.validator.apply_to_ddl property is set to true.

Reply:
https://www.baeldung.com/hibernate-notnull-vs-nullable

Reply:
Nevermind validation before db will take place in service so @NotNull would be redundant.